src/themes/teams/components/Attachment/attachmentStyles.ts

```diff
@@ -0,0 +1,69 @@
+import { IComponentPartStylesInput, ICSSInJSStyle } from '../../../../../types/theme'
+import { AttachmentProps } from '../../../../components/Attachment/Attachment'
+import { AttachmentVariables } from './attachmentVariables'
+import { pxToRem } from '../../../../lib'
+import { Extendable } from '../../../../../types/utils'
+
+type StyleParam = { props: Extendable<AttachmentProps>; variables: AttachmentVariables }
+
+const attachmentStyles: IComponentPartStylesInput = {
+  root: ({ props, variables }: StyleParam): ICSSInJSStyle => ({
+    position: 'relative',
+    display: 'inline-flex',
+    alignItems: 'center',
+    width: pxToRem(300),
+    minHeight: pxToRem(48),
+    padding: pxToRem(8),
+    marginBottom: pxToRem(2),
+    marginRight: pxToRem(2),
+    background: variables.backgroundColor,
+    color: variables.textColor,
+
+    ...((props.actionable || props.onClick) && {
+      cursor: 'pointer',
+
+      ':hover': {
+        background: variables.backgroundColorHover,
+      },
+    }),
+  }),
+
+  content: ({ props }: StyleParam): ICSSInJSStyle => ({
+    flex: 1,
+  }),
+
+  header: ({ props, variables }: StyleParam): ICSSInJSStyle => ({
+    fontSize: variables.headerFontSize,
+    fontWeight: variables.headerFontWeight,
+    lineHeight: variables.headerLineHeight,
+  }),
+
+  description: ({ props, variables }: StyleParam): ICSSInJSStyle => ({
+    opacity: 0.5,
+    fontSize: variables.descriptionFontSize,
+    fontWeight: variables.descriptionFontWeight,
+    lineHeight: variables.descriptionLineHeight,
+  }),
+
+  icon: ({ props, variables }: StyleParam): ICSSInJSStyle => ({
+    flex: '0 0 auto',
+    marginRight: variables.iconSpace,
```

Comment:
I think we don't want this margin since the attachment's 8px padding is enough to get this icon in the correct place.

Reply:
This is between the icon and the header/description.  I believe the redline called for a larger value here.